dicomImport/matRad_importDicomRTPlan.m

```diff
@@ -56,21 +56,26 @@
     errordlg('Not supported kind of DICOM RT plan file.');    
 end
 
-% use the treatment beams only
+% get beam sequence
 BeamSequence = planInfo.(BeamParam);
 BeamSeqNames = fieldnames(BeamSequence);
-for i = 1:length(BeamSeqNames)
-    currBeamSeq = BeamSequence.(BeamSeqNames{i});
-    try
-        treatDelType = currBeamSeq.TreatmentDeliveryType;
-        if ~strcmpi(treatDelType,'TREATMENT')
-            BeamSequence = rmfield(BeamSequence,BeamSeqNames{i});
+
+% use the treatment beams only
+if isfield(BeamSequence.(BeamSeqNames{1}),'TreatmentDeliveryType')
+    for i = 1:length(BeamSeqNames)
```

Comment:
I do not understand why we need this check

if isfield(BeamSequence.(BeamSeqNames{1}),'TreatmentDeliveryType')

Shouldn't this just fail within the try catch statement in line 68? And: What happens if you dont have the field 'TreatmentDeliveryType' in BeamSeqNames{1} but maybe in BeamSeqNames{2} or BeamSeqNames{3}? Then you lose all beams right here!